src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -113,6 +114,7 @@ public enum UploadErrorType
         private static string serverUrl;
         private static string page;
         private readonly Regex serverResponceRegex;
+        private readonly CameraData cameraData;
```

Comment:
Is there a strong reason to store this information? Where possible, we should try to avoid storing intermediate state in fields when we could pass it as a parameter. This prevents us from running into issues where old state is erroneously hanging around.